bin.js

```diff
@@ -3,49 +3,166 @@
 const ipfsProvider = require('ipfs-provider')
 const meow = require('meow')
 const cohost = require('.')
+const ora = require('ora')
+const prettyBytes = require('pretty-bytes')
+
+let log = null
+let spin = null
+
 const cli = meow(`
   Usage
-    $ ipfs-cohost <domain 1> <domain 2>...
+    $ ipfs-cohost <domain>...
+    $ ipfs-cohost add <domain>... [--lazy] [--full]
+    $ ipfs-cohost rm <domain>...
+    $ ipfs-cohost ls [domain]...
+    $ ipfs-cohost mv [domain]... [--lazy] [--full]
+    $ ipfs-cohost sync
+    $ ipfs-cohost prune [n]
 
   Example
     $ ipfs-cohost docs.ipfs.io cid.ipfs.io
 
   Options
-    --no-pin, Find the cumlative sizes but dont pin them
     --silent, -s  Just do your job
+    --full        Fully cohost a website
+    --lazy        Lazily cohost a website (default)
```

Comment:
Why is lazy the default? Most websites aren't super big so personally I'd be ok with co-hosting them in full. Also, we want to encourage cohosting right? For people who don't take the time to read the help they'll just `ipfs-cohost add wikipedia` and bam they think they're hosting the whole thing when actually they're not.

Reply:
We have an issue about that: https://github.com/ipfs-shipyard/cohosting/issues/6, please take a look.

Reply:
Personally, I wouldn't define the default for this in the spec. There's going to be different use cases and situations for lazy vs full. I think for a command line utility where I'm explicitly adding sites I want to cohost the default should be full, since actually lazily loading a site would require me to use a _different_ application to explore it and start to lazily load bits of it. `ipfs-cohost add wikipedia` is effectively useless unless I then fire up my browser, load up _my_ gateway, navigate to the site I just decided to cohost, and then start navigating around.

I totally get that in companion you'd want to set this to lazy by default, because it transparently redirects IPFS sites and it's likely to be more of an ambient setting than an explicit COHOST THIS NOW type of action.

cc @lidel

Reply:
>  I think for a command line utility where I'm explicitly adding sites I want to cohost the default should be full [..]

I agree the CLI expectation is "full", but English Wikipedia is 650GB and we have no API to track progress of `refs -r` once `add --full` is executed :(
People may not understand why `add` takes days to complete.

Some ideas:

- **(A)** default CLI to "full", but inform user about the size of added website, giving an option to cancel before fetch starts
- **(B)** default CLI to "full", but fail if added website is more than X Megabytes and ask user to pass explicit `--full` to indicate they are ok with the cost
- **(C)** return an error on `add` without explicit `--lazy` or `--full` (require explicit decision)

I think I like **(B)** the most, but perhaps there is a simpler way to address this?

Reply:
Yes, I agree that `--full` may become the default but we should warn if it's bigger than X. So B for me.

Reply:
B sounds great!